packages/@react-stately/dnd/src/useDraggableCollectionState.ts

```diff
@@ -65,7 +82,8 @@ export function useDraggableCollectionState(props: DraggableCollectionStateOptio
     onDragMove,
     onDragEnd,
     preview,
-    getAllowedDropOperations
+    getAllowedDropOperations,
+    getPreviewOffset
```

Comment:
I know this is just gets passed through, but can you add a story or a test just making sure that providing this to useDragAndDrop works?